manim/mobject/coordinate_systems.py

```diff
@@ -699,6 +706,482 @@ def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
+        return lines1, lines2
+
+    def get_center_point(self):
+        return self.coords_to_point(0, 0)
+
+    def get_x_unit_size(self):
+        return self.get_x_axis().get_unit_size()
+
+    def get_y_unit_size(self):
+        return self.get_x_axis().get_unit_size()
+
+    def get_axes(self) -> VGroup:
+        """Gets the axes.
+        Returns
+        -------
+        :class:`~.VGroup`
+            A pair of axes.
+        """
+        return self.axes
+
+    def get_vector(self, coords, **kwargs):
+        kwargs["buff"] = 0
+        return Arrow(
+            self.coords_to_point(0, 0), self.coords_to_point(*coords), **kwargs
+        )
+
+    def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
+        for mob in self.family_members_with_points():
+            num_curves = mob.get_num_curves()
+            if num_inserted_curves > num_curves:
+                mob.insert_n_curves(num_inserted_curves - num_curves)
+        return self
+
+    def polar_to_point(self, radius: float, azimuth: float) -> np.ndarray:
+        r"""Gets a point from polar coordinates.
+
+        Parameters
+        ----------
+        radius
+            The coordinate radius (:math:`r`).
+
+        azimuth
+            The coordinate azimuth (:math:`\theta`).
+
+        Returns
+        -------
+        numpy.ndarray
+            The point.
+
+        Examples
+        --------
+
+        .. manim:: PolarToPointExample
+            :ref_classes: PolarPlane
+            :save_last_frame:
+
+            polarplane_pi = PolarPlane(azimuth_units="PI radians", size=6)
+            polartopoint_vector = Vector(polarplane_pi.polar_to_point(3, PI/4))
+
+            class PolarToPointExample(Scene):
+                def construct(self):
+                    self.add(polarplane_pi)
+                    self.add(polartopoint_vector)
+        """
+        return self.coords_to_point(radius * np.cos(azimuth), radius * np.sin(azimuth))
+
+    def pr2pt(self, radius: float, azimuth: float) -> np.ndarray:
+        """Abbreviation for :meth:`polar_to_point`"""
+        return self.polar_to_point(radius, azimuth)
+
+    def point_to_polar(self, point: np.ndarray) -> Tuple[float, float]:
+        r"""Gets polar coordinates from a point.
+
+        Parameters
+        ----------
+        point
+            The point.
+
+        Returns
+        -------
+        Tuple[:class:`float`, :class:`float`]
+            The coordinate radius (:math:`r`) and the coordinate azimuth (:math:`\theta`).
+        """
+        x, y = self.point_to_coords(point)
+        return np.sqrt(x ** 2 + y ** 2), np.arctan2(y, x)
+
+    def pt2pr(self, point: np.ndarray) -> Tuple[float, float]:
+        """Abbreviation for :meth:`point_to_polar`"""
+        return self.point_to_polar(point)
+
+    def get_coordinate_labels(
+        self,
+        r_values: Optional[Iterable[float]] = None,
+        a_values: Optional[Iterable[float]] = None,
+        **kwargs,
```

Comment:
Wouldn't it make more sense if `azimuth_label_buff` and `azimuth_label_scale` + relevant label parameters were defined here instead?
(also for `add_coordinates`)

Reply:
I put them in the class constructor because the radius scale and buff are defined by `radius_config` which goes into the super `axis_config`. Since the azimuth labels are not part of a `NumberLine` they have to be drawn separately, but the radius labels are drawn in the same way as `NumberPlane`. I could move them to `add_coordinates` but currently it would mean defining the radius label parameters in the constructor and the azimuth label parameters in `add_coordinates` which would obviously be awkward.

Reply:
Seems fair.